### PR TITLE
Add Amphetamine Enhancer.app :latest

### PR DESCRIPTION
**Pull request by revonoc**
The app requires Amphetamine 5.0 from Mac App Store and is made by the same person to add functionality

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

**Comment by revonoc** [comment]
I'm a noob and have no idea why this is failing checks - I did the style check already but am not sure what to do about the other fails. Also the app autoupdates

File: Casks/amphetamine-enhancer.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'amphetamine-enhancer' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://github.com/x74353/Amphetamine-Enhancer/raw/master/Releases/Current/Amphetamine%20Enhancer.dmg'
+  name 'Amphetamine Enhancer'
+  homepage 'https://github.com/x74353/Amphetamine-Enhancer'
+
```
Comment (miccal) [inline Casks/amphetamine-enhancer.rb:8]: You will to add an `app` stanza:
```
app 'Amphetamine Enhancer.app'
```

File: Casks/amphetamine-enhancer.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  caveats 'requires Amphetamine 5.0+ from Mac App Store'
+  caveats 'uninstall any scripts within the app before uninstalling'
+  caveats 'uninstall any scripts within the app before uninstalling'
```
Comment (miccal) [inline Casks/amphetamine-enhancer.rb:11]: Add `caveats` like so:
```
  caveats <<~EOS
    Caveats text goes here.
  EOS
```

Comment (revonoc) [reply Casks/amphetamine-enhancer.rb]: i think i did so correctly, thank you!

File: Casks/amphetamine-enhancer.rb (hunk 1 above)
Comment (revonoc) [reply Casks/amphetamine-enhancer.rb:8]: thank you!

**Comment by miccal** [comment]
Please fix the following errors from `brew cask style`:
```
==> brew cask style amphetamine-enhancer
== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks/amphetamine-enhancer.rb ==
C:  9:  3: caveats stanza out of order
C: 13:  1: stanza groups should be separated by a single empty line
C: 13:  1: Trailing whitespace detected.
C: 14:  3: app stanza out of order
```

**Comment by revonoc** [comment]
woop woop

**Comment by vitorgalvao** [comment]
Thank you for the submission, and sorry for refusing after you’ve done the work and corrections, but we’re increasing our notability requirements to match those of Homebrew (30 forks, 30 watchers, 75 stars), which this doesn’t pass.

Also, it’s not very common for us to add software that requires a MAS version as well, and being unversioned and not having an appcast hurts the consideration to include this cask, because it makes it harder to maintain and include in upgrades.

**Pull request closed without merging**